CONTRIBUTING.md

```diff
@@ -50,11 +50,11 @@ impact the functionality of `conda/conda-build` installed in your base environme
 
 ``` bash
     # create/activate standalone dev env
-    $ conda create --name dev --file tests/requirements.txt --channel defaults
-    $ conda activate dev
+    $ CONDA_ENV=conda-build make setup
+    $ conda activate conda-build
 
-    # run tests
-    (dev) $ pytest
+    # Run all tests (can take a long time)
+    (dev) $ make test
```

Comment:
```suggestion
    # Run all tests on Linux and Mac OS X systems (this can take a long time)
    (dev) $ make test
```

Reply:
**Note:** Since we're changing the suggested env name from `dev` to `conda-build`, any instances of `(dev)` (where the env name is displayed in a command line example) will need to change to `(conda-build)` for consistency